index.js

```diff
@@ -1 +1,101 @@
 // implement your API here
+const express = require("express");
+const db = require("./data/db.js");
+const server = express();
+server.use(express.json());
+server.use(cors());
+
+server.post("/api/users", (req, res) => {
```

Comment:
great job on the logic for the core concept of this assignment and using the correct http codes, and accounting for the required situtations for successes, failures and everything.